pkg/platform/provider/baremetal/phases/authzwebhook/authzwebhook.go

```diff
@@ -58,11 +59,35 @@ type Option struct {
 	IsGlobalCluster      bool
 }
 
+// WebhookCertAndKeyExist checks whether the certificate and private key exist,
+// for compatibility with old version clusters' webhook certificates and private keys.
```

Comment:
please highlight this is involved for tke1.5 and will get removed from tke1.6 or future